### PR TITLE
Bug/preview reconnection

### PR DESCRIPTION
**Pull request by Decoder07**
Changes:
- Changed scale_type from FIT to FILL

### CODE REVIEW

File: android/src/main/kotlin/live/hms/hmssdk_flutter/views/HMSVideoViewFactory.kt [hunk 1]
```diff
@@ -41,8 +41,9 @@ class HMSVideoViewWidget(context: Context, id: Int, creationParams: Map<String?,
         view.layoutParams = frameLayoutParams
 
         if (peer == null) return
-
         val tracks = peer.auxiliaryTracks
+        
+        Log.i("HMSVideoView","${peer.name} ${isAux} ${tracks.isNotEmpty()}")
```
Comment (jashan1498) [inline android/src/main/kotlin/live/hms/hmssdk_flutter/views/HMSVideoViewFactory.kt:46]: remove

File: example/lib/preview/preview_controller.dart [hunk 2]
```diff
@@ -15,8 +15,8 @@ class PreviewController {
         await RoomService().getToken(user: user, room: roomId);
 
     if (token == null) return false;
+    if (token[0] == null) return false;
```
Comment (jashan1498) [inline example/lib/preview/preview_controller.dart:18]: can crash on empty list

File: lib/src/ui/meeting/hms_video_view.dart [hunk 3]
```diff
@@ -83,7 +83,7 @@ class _PlatformView extends StatelessWidget {
           'track_id': track.trackId,
           'is_aux': track.source != "REGULAR",
           'screen_share': track.source != "REGULAR",
-          'scale_type': ScalingType.SCALE_ASPECT_FIT.value,
+          'scale_type': ScalingType.SCALE_ASPECT_FILL.value,
```
Comment (jashan1498) [inline lib/src/ui/meeting/hms_video_view.dart:86]: reason for change ?

**Review by Arunshaik2001 — APPROVED** [review]

File: example/lib/preview/preview_controller.dart (hunk 2 above)
Comment (Decoder07) [reply example/lib/preview/preview_controller.dart:18]: how ?

**Pull request merged**